setup.py

```diff
@@ -36,7 +36,8 @@
     package_dir={'': 'src'},
     install_requires=[
         'async_generator',
-        'trio >= 0.11.0'
+        'trio >= 0.11.0',
```

Comment:
Note that trio-util supports old, pre-ExceptionGroup versions of trio.  As the primary user of trio-util, my own organization is still on trio 0.21.  Effectively, this PR is disallowing trio < 0.22.